src-vue/src/views/SettingsView.vue

```diff
@@ -14,6 +14,17 @@
                         <el-button icon="Folder" @click="updateGamePath"/>
                     </template>
                 </el-input>
+
+                <!-- Thunderstore mods per page configuration -->
+                <div class="fc_parameter__panel">
+                    <h3>Number of Thunderstore mods per page</h3>
+                    <h6>This has an impact on display performances when browsing Thunderstore mods.</h6>
+                    <el-input 
+                        v-model="modsPerPage" 
+                        type="number"
+                    ></el-input>
```

Comment:
A "_Reset to default_" button next to the field would be nice to have. Could also decrease the length of the input field to fit the button without increasing the width of the overall page.

There's a lot of empty space here anyway ^^
![image](https://user-images.githubusercontent.com/40122905/210553833-b73d480a-3296-4574-8b4c-23026a2fa801.png)

Reply:
Addressed in 388113f450d4f933789a7a0c0a696462f1e2ee10